cypress.config.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+const { defineConfig } = require('cypress');
+
+module.exports = defineConfig({
+  defaultCommandTimeout: 60000,
+  requestTimeout: 60000,
+  responseTimeout: 60000,
+  baseUrl: 'http://localhost:5601',
+  viewportWidth: 2000,
+  viewportHeight: 1320,
+  env: {
+    openSearchUrl: 'http://localhost:9200',
+    SECURITY_ENABLED: false,
+    AGGREGATION_VIEW: false,
+    username: 'admin',
+    password: 'myStrongPassword123!',
+    ENDPOINT_WITH_PROXY: false,
+    MANAGED_SERVICE_ENDPOINT: false,
+    VISBUILDER_ENABLED: true,
+    DATASOURCE_MANAGEMENT_ENABLED: false,
+    ML_COMMONS_DASHBOARDS_ENABLED: true,
+    WAIT_FOR_LOADER_BUFFER_MS: 0,
```

Comment:
It seems only `openSearchUrl` is needed based on the migrated test?

Reply:
Yes but as the plan is to right away start writing new tests in this repo, I've added/matched mostly what is in FTRepo.